lms/djangoapps/instructor/services.py

```diff
@@ -36,7 +37,7 @@ def delete_student_attempt(self, student_identifier, course_id, content_id, requ
             - content_id is a url-name of a problem
             - course_id is the id for the course
         """
-        course_id = CourseKey.from_string(course_id)
+        course_id = course_key_from_string_or_404(course_id)
```

Comment:
is this an appropriate place to use this helper? Is it only being called by a view?